attributes/default.rb

```diff
@@ -33,6 +33,8 @@
 default['onlinefs']['monitoring']             = 12800
 
 default['onlinefs']['service']['thread_number'] = 10
```

Comment:
is this still needed, right now it looks like it is used for metrics only. why dont we make it `ron_db_thread_number` + `vector_db_thread_number` in onlinefs.